content/writing-npm-packages.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Writing npm Packages
+order: 31
+discourseTopicId: 20194
+---
+
+**NOTE:** If you want to override a package that already exists in the npm registry, [use this method](#overriding-npm-packages).
```

Comment:
I wonder why this is the first thing in the article - if it is in the table of contents I suspect people will find it!